src/App.js

```diff
@@ -71,65 +49,90 @@ class App extends Component {
     tubeMap.setTransparentNodesFlag(visOptions.transparentNodes);
     tubeMap.setShowReadsFlag(visOptions.showReads);
     tubeMap.setSoftClipsFlag(visOptions.showSoftClips);
-    tubeMap.setColorSet('haplotypeColors', visOptions.haplotypeColors);
-    tubeMap.setColorSet('forwardReadColors', visOptions.forwardReadColors);
-    tubeMap.setColorSet('reverseReadColors', visOptions.reverseReadColors);
+    tubeMap.setColorSet("haplotypeColors", visOptions.haplotypeColors);
+    tubeMap.setColorSet("forwardReadColors", visOptions.forwardReadColors);
+    tubeMap.setColorSet("reverseReadColors", visOptions.reverseReadColors);
     tubeMap.setColorReadsByMappingQualityFlag(
       visOptions.colorReadsByMappingQuality
     );
     tubeMap.setMappingQualityCutoff(visOptions.mappingQualityCutoff);
   }
+  /*
+   * Drop undefined values
+   * See https://stackoverflow.com/questions/286141/remove-blank-attributes-from-an-object-in-javascript/38340730#38340730
+   */
+  removeUndefined = (obj) => {
+    return Object.fromEntries(
+      Object.entries(obj).filter(([_, v]) => v != null)
+    );
+  };
+
+  /**
+   * @param {ViewTarget} viewTarget - The default or selected data to view
```

Comment:
This doc comment should also explain how the "current" view target is the one displayed in the tube map itself.